### DISCUSSION TITLE
Tasmota - WebUI - insa(array)

### DISCUSSION THREAD

**Discussion by KKlapper** [arendst/Tasmota#17254] [Q&A] [answered]
Hi all, I tried to customize the WebUI of Tasmota. 

Does someone know the syntax of insa(array)? It is documented, but it does not work for me
    "insa(array) in any position insert all elements from an array comma separated"

insa(array) is display as string in the >W section, but it does not dump values of the array.

Thanks

**Comment by gemu2015** [comment, marked as answer]
inserts a comma separated array
e.g.

data: [insa(array)]

it needs
#define USE_GOOGLE_CHARTS
